### PR TITLE
Fixes #52

### LINKED ISSUE #52: Feather RP2040 RFM69 Packet Radio version 35 not supported
A recently-purchased feather has an rfm69 device that reports version 0x23 which fails a version check in the RFM69 constructor:

```
...
  File "/lib/adafruit_rfm69.py", line 316, in __init__
RuntimeError: Invalid RFM69 version, check wiring!
```

Version number extraction from REPL:
```
import time
import board
import digitalio
import adafruit_bus_device.spi_device as spidev
from micropython import const

reset = digitalio.DigitalInOut(board.RFM_RST)
reset.switch_to_output(value=False)
reset.value = True
time.sleep(0.0001)
reset.value = False
time.sleep(0.005)

buff = bytearray(4)
dev = spidev.SPIDevice(board.SPI(), digitalio.DigitalInOut(board.RFM_CS), baudrate=2000000, polarity=0, phase=0)
with dev as d:
  buff[0] = const(0x10) & 0x7f
  d.write(buff, end=1)
  d.readinto(buff, end=4)


print(f"{buff[0]:#0x}")
```

### PR DESCRIPTION
**Pull request by michaeldye**

### CODE REVIEW

File: adafruit_rfm69.py [hunk 1]
```diff
@@ -312,7 +312,7 @@ def __init__(  # pylint: disable=invalid-name
         self.reset()  # Reset the chip.
         # Check the version of the chip.
         version = self._read_u8(_REG_VERSION)
-        if version != 0x24:
+        if version not in [0x23, 0x24]:
```
Comment (dhalbert) [inline adafruit_rfm69.py:315]: Will use a tuple here. Might be slightly more efficient.

```suggestion
        if version not in (0x23, 0x24):
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks for this fix!

**Review by dhalbert — APPROVED** [review]

**Review by ladyada — APPROVED** [review]
thanks!

**Comment by dhalbert** [comment]
Incorporated into new release https://github.com/adafruit/Adafruit_CircuitPython_RFM69/releases/tag/2.1.20

**Pull request merged**